Changes

```diff
@@ -1,3 +1,167 @@
+- Removed ProceduralHolder node. This was not exposed via the UI,
+  had never been used, and is no longer supported by any of the
+  renderer backends (#2366).
+- Shader : Removed deprecated state methods (#2354).
+- Removed RenderableGadget (#2334). If similar functionality is required,
+  use an ObjectToSceneNode and a SceneGadget.
+- SceneAlgo : Removed camera/transform utilities (#2332). These relied
+  on conventions and functionality that will be removed in Cortex 10.
+- RendererAlgo : Removed obsolete functions used only with the legacy
+  renderer backends (#2318).
+- Removed code for handling `IECore::Light` (#2331). Lights are now
+  always represented as shaders assigned to a location.
+- Removed SceneProcedural and ScriptProcedural. These were only useful
+  in legacy renderer backends, which have now all been removed (#2318).
+- Removed ExecutableRender node (#2318). This used the legacy renderer backends
+  which have all been removed.
+- Gadget : Modified rendering API (#2304).
+- Removed IncrementingPlugValueWidget (#2322). Use RefreshPlugValueWidget
+  instead.
+- Removed EnumPlugValueWidget (#2322). Use PresetsPlugValueWidget and
+  metadata presets instead.
+- ViewportGadget : Refactored camera transform handling (#2351). The camera
+  transform is now accessed via new `setCameraTransform()`/`getCameraTransform()`
+  methods, rather than via the camera itself.
+- ScriptNode : Modified signature of `ScriptExecutedSignal` to use raw pointers
+  (#2336).
+
+Features
+--------
+
+- Arnold 5 support (#2308) :
+  - Added support for rendering arbitrary meshes as volumes. Previously
+    they were converted to box volumes.
+  - Added support for light path expressions.
+  - Added support for transform_type.
+  - Added ArnoldAOVShader node to support AOV shaders.
+  - Added support for volume_padding (#2356).
+- 3Delight 13 support via the NSI API (#2311).
+  - All shading/lighting is now performed via OSL shaders.
+  - Includes full support for adding/removing/moving objects during
+    interactive rendering.
+- Appleseed 1.8 support (#2371) :
+  - Updated version to 1.8.1.
+  - Added AOV presets to the Outputs node.
+- VDB (#2373) :
+  - Added native support for representing VDB volume data in Gaffer
+    scenes.
+  - The standard SceneReader node can now load .vdb files.
+  - VDB objects can be rendered directly to Arnold.
+  - The Viewer now provides basic visualisation of VDB grids.
+  - The SceneInspector provides inspection of grid metadata etc
+    in the Object section.
+  - Several new nodes demonstrate the feasibility of manipulating
+    VDB grids directly in Gaffer's node graph :
+      - LevelSetToMesh
+      - MeshToLevelSet
+      - LevelSetOffset
+- Catalogue : Added a new CatalogueSelect node. This can be placed
+  downstream of a Catalogue to override the image to view (#2370).
+- UI Editor : Added the ability to add custom buttons to a plug (#2348).
+- Gadget : Introduced layer-based rendering for Gadget-based UI
```

Comment:
Could be more user friendly, mentioning that plug labels draw on top now?

Reply:
Oh, I guess that is mentioned further down under Fixes... seems a bit buried down there.